pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -2643,6 +2643,9 @@ in {
   # 2022-10-04: Needs newer tasty-dejafu than (currently) in stackage
   rec-def = super.rec-def.override { tasty-dejafu = self.tasty-dejafu_2_1_0_0; };
 
+  # 2022-11-05: https://github.com/ysangkok/haskell-tzdata/issues/3
+  tzdata = dontCheck super.tzdata;
```

Comment:
Should be fixed as soon as https://github.com/NixOS/nixpkgs/pull/198475 propagates to our branch. Possibly should have an `assert pkgs.tzdata.version == "2022e"; … ` in front.